infrastructure/src/main/java/org/corfudb/infrastructure/NodeView.java

```diff
@@ -0,0 +1,55 @@
+package org.corfudb.infrastructure;
+
+import lombok.Data;
+import org.corfudb.format.Types.NetworkMetrics;
+import org.corfudb.format.Types.ClusterView;
+
+
+/**
+ * NodeView:
+ * <p>
+ * Contains the network status of the state of a node and
```

Comment:
This comment is not easy to parse. Could you please reword it so that it is easier to understand.